data/filters/templates/youtube-homepage-cleanup.yaml

```
@@ -0,0 +1,39 @@
+title: "YouTube: homepage interface cleanups"
+contributors:
+  - JohnyP36
+  - xvello
```

I didn't contribute anything (yet) as I don't count reviews, I'll add myself when relevant.
```suggestion
  - JohnyP36
```